frontend/packages/core/src/AppProvider/workflow.tsx

```diff
@@ -47,6 +47,8 @@ interface Route {
   path: string;
   /** Properties required by the Component that are set only via the config. */
   requiredConfigProps?: string[];
+  /** Is the workflow discoverable via search and drawer navigation. This defaults to false. */
+  private?: boolean;
```

Comment:
would `hideNav` be a better var? it's not really private.

Reply:
Sure. `discoverable`, `hidden`, `hideNav`?